modules/oe_theme_helper/tests/src/Unit/TwigExtensionTest.php

```diff
@@ -262,18 +262,20 @@ protected static function getEuropeanUnionLanguageList(): array {
    *
    * @param string $icon_name
    *   The icon name.
+   * @param string|null $size
+   *   The icon size.
    * @param array $expected_icon_array
    *   The icon array to be rendered.
    *
    * @covers ::toEclIcon
    * @dataProvider toEclIconProvider
    */
-  public function testToEclIcon(string $icon_name, array $expected_icon_array) {
+  public function testToEclIcon(string $icon_name, string $size = NULL, array $expected_icon_array) {
     $context = ['ecl_icon_path' => '/path/to/theme/resources/icons/'];
     // We join the resulting array from to_ecl_icon() function so that we have
     // a visual representation of the array being returned by the function.
-    $result = $this->twig->render("{{ to_ecl_icon('$icon_name')|join('|') }}", $context);
-    $this->assertEquals(implode('|', $expected_icon_array), $result);
+    $result = $this->twig->render("{{ to_ecl_icon('$icon_name', '$size')|join('|') }}", $context);
+    $this->assertEquals(implode('|', array_filter($expected_icon_array)), $result);
```

Comment:
We don't test when we don't pass any size to the function, like now we test `to_ecl_icon('$icon_name', '')` but never `to_ecl_icon('$icon_name')`, let's add an if and, if `$size === NULL`, then we use `to_ecl_icon('$icon_name')` in our test, like that we can test both cases, when not passed or when passed empty (it could happened if size is a variable coming from somewhere else).